orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/pipeline/cluster/RollbackClusterStage.java

```diff
@@ -89,21 +91,19 @@ public void afterStages(@Nonnull Stage parent, @Nonnull StageGraphBuilder graph)
       // propagate any attributes of the parent stage that are relevant to this rollback
       context.putAll(propagateParentStageContext(parent.getParent()));
 
-      Stage rollbackStage = graph.add((it) -> {
+      graph.append((it) -> {
```

Comment:
This was incorrect. Using `add` would create a stage with no `requisiteStageRefIds` when what's intended here is to run each region in serial. Spotted it when updating stages to use the new `append` method.

Reply:
👍